### DISCUSSION TITLE
Publishing a third edition of the Leaders Manual

### DISCUSSION THREAD

**Discussion by semioticrobotic** [open-organization-ambassadors/open-org-leaders-manual#51] [Ideas]
On a recent community call, a few interested community members [discussed the prospect](https://github.com/open-organization/governance/blob/master/meeting-notes/2024/2024-05-20-meeting-notes.md#next-edition-of-leaders-manual) of a third edition of *The Open Organization Leaders Manual*. We generally considered it a Good Idea™ and decided to sketch some ideas for what it might look like, should we decide to create such a thing.

The primary impetus for writing a third edition of the manual is a desire for it to reflect the ways the Open Organization project's thinking on the topic of open leadership has evolved since it issued the second edition. We now have a concretely articulated [Open Leadership Definition](https://github.com/open-organization/open-leadership-definition), for example—which did not exist at the time of previous publication. We also have a fully featured [Open Leadership Assesment](https://github.com/open-organization/open-leadership-assessment)—also something that did not exist when we issued prior editions. And moreover (perhaps most importantly), many community members have been speaking, writing, and working in the field in the (too many) years since prior publication. Our book should capture iterations in their practice and their guidance.
 
Historically, our first step in this kind of work has been to sketch a proposed table of contents. Doing this allows us to both catalog the chapters we have and to identify opportunities to add new voices to the mix. I volunteered to review our [existing conents](https://github.com/open-organization/open-org-leaders-manual/blob/master/CONTENTS.md) and rearrage our materials into a new organizational schema. The second edition of the manual is organized into three sections:

* Planning & Goal Setting
* Organizational Design and Culture Building
* Motivation and Engagement

During recent discussions, we decided to experiment with re-organizing the book according to the characteristics outlined in the Open Leadership Definition:

* Transparency
* Inclusivity
* Collaboration
* Adaptability
* Community

In what follows, then, I will offer one possible reshuffle of existing materials. This is simply a starting point for our joint interations. I suspect folks will have other (better!) opinions on how we might organize all this.

### Transparency
Doyle: Setting goals transparently and collaboratively
Sjölund: Visualizing your plans and progress
Groen: Organizational transformation through open leadership
Hall: Open leadership lessons from a galaxy far, far away

### Inclusivity
Hilliger & Matlack: Owning your career in an open organization
Niveyro: Becoming a leader with inclusive awareness
Carver: Leading through the power of "thank you"
Groen: Transformation and leadership in a hybrid world
Burchard: More engaging meetings begin with trustful relationships

### Collaboration
Robertson: An open leader's guide to better meetings
McFarland: When empowering employee decision-making, intent is everything
Matlack: The Tao of project management  

### Adaptability
Knuth: Planning for the future isn't what it used to be
Louis: Creating teams that aren't afraid to fail
Louis: Developing a culture of experimentation on your team
Hess von Ludewig: Making cultural transformation manageable
Hess von Ludewig: Creativity is risky (and other truths open leaders need to hear)

### Community
Alexander: When open values drive open behaviors: Defining open leadership
Whitehurst: What it means to be an open leader
Kelchner: Stop hiring for culture fit
Kelchner: To survive Industry 4.0, leaders should think beyond the digital
Fattorini: Innovation through community
Sansing: Let engagement lead the way

-----

Our next goals will be to decide how to handle each piece: whether to **reprint** it as-is, to **rewrite** it and publish a new version, or **drop** it entirely in favor of something else. To help us make those decisions, I'll lay out all the chapters in a table and ask authors featured in the manual to let us know what they'd like to do. Where possible, I've tagged them into the discussion to get their attention more directly.

I'll start, using my preface as an example.

| Chapter | Author | Reprint? | Rewrite? | Drop? |
| ------- | ------ | -------- | -------- | ----- |
| Preface | @semioticrobotic | -- | ✓ | -- |
| Planning for the future isn't what it used to be | @samuelknuth | -- | -- | ✓ |
| Creating teams that aren't afraid to fail | @cll | -- | ✓ | -- |
| Developing a culture of experimentation on your team | @cll | -- | ✓ | -- |
| Owning your career in an open organization | @amatlack and @LauraHilliger | -- | ✓ | -- |
| The Tao of project management | @amatlack | ✓ | -- | -- |
| Setting goals transparently and collaboratively | Doyle | -- | -- | -- |
| Visualizing your plans and progress | @jimmysjolund | -- | ✓ | -- |
| When open values drive open behaviors: Defining open leadership | Alexander | -- | -- | -- |
| Stop hiring for culture fit | @jenkelchner | -- | -- | -- |
| Innovation through community | Fattorini | -- | -- | -- |
| What it means to be an open leader | Whitehurst | -- | -- | -- |
| An open leader's guide to better meetings | @arob98 | -- | -- | -- |
| Making cultural transformation manageable | @HeidiHVL | -- | -- | -- |
| Creativity is risky (and other truths open leaders need to hear) | @HeidiHVL | -- | -- | -- |
| Becoming a leader with inclusive awareness | Niveyro | -- | -- | -- |
| Organizational transformation through open leadership | @JosGroenNL | -- | -- | -- |
| Transformation and leadership in a hybrid world | @JosGroenNL | -- | -- | -- |
| Let engagement lead the way | @chadsansing | -- | -- | -- |
| When empowering employee decision-making, intent is everything | @RonMcFarland | -- | -- | ✓ |
| Leading through the power of "thank you" | Carver | -- | -- | ✓ |
| More engaging meetings begin with trustful relationships | Burchard | -- | -- | -- |
| Open leadership lessons from a galaxy far, far away | @freedosproject | -- | ✓ | -- |

-----

Of course, we wouldn't just reshuffle the chapters and reprint. We'd ask authors to review their work and perhaps adjust it to fit the theme of their newly appointed section if necessary. We'd also want to address [outstanding issues](https://github.com/open-organization/open-org-leaders-manual/issues) with the second edition. And most important of all: We'll want to invite **new voices** to join the conversation.

Before we can do any of that, however, we've got to take stock. Hence the above.

🤔 So what do folks think? Does this sound like something our authors might be interested in?

**Comment by jimmysjolund** [comment]
Visualizing your plans and progress - Rewrite. Ongoing in https://github.com/open-organization/open-org-leaders-manual/issues/50

**Comment by cll** [comment]
I think it is a great idea. Question: If we make any edits, where would you
like us to source the edits?

Thank you,

Catherine

**Reply by semioticrobotic** [reply]
Glad you're up for it, @cll! And this is a great question.

To get started, I'd recommend opening an issue describing the work you'd like to do. This gives us a little more information we can use to triage the changes and set [our project plan](https://github.com/orgs/open-organization/projects/4).

Then you can open a pull request to start drafting the changes, and editors can work with you there to iterate as you go.

Sound good?

**Comment by RonMcFarland** [comment]
Bryan,
If I am selected for an article on Collaboration, I think a more recent article might be better.  You mentioned my very first article I posted in 2016.  

I would recommend my most recent article on collaboration and making judgements.  "Reduce noise to make better decisions"  (https://coachingbuttons.com/2023/12/25/reducenoise), Use this process to reduce the noise that creates radically different judgments in group collaboration.  December 25, 2023

**Reply by semioticrobotic** [reply]
Thanks, @RonMcFarland. Whatever you think is best is, I'm sure, best here. I will make note of this in the project plan.

**Reply by RonMcFarland** [reply]
Sounds good Bryan.  We can make a final decision on this when the full content is laid out and see how all the article fit together.  I noticed I have nine articles that touch on collaboration.  So, we can pick another one if it is more appropriate.

**Reply by RonMcFarland** [reply]
Also, combining the two articles is a good idea.  We can hit the most important points within each of the two.

**Comment by samuelknuth** [comment]
I think this is a great idea @semioticrobotic - thank you for including me.
I would be open to contributing something for sure. I haven't re-read my
article, but from the title I suspect it's quite dated, and it's also a
topic I no longer actively work in. I think the most effective contribution
would be if I wrote something about communication, addressing conflict, or
difficult conversations (like sharing negative performance feedback).

**Reply by semioticrobotic** [reply]
I'd love to read some new writing from you, @samuelknuth. Yes, please—give it a thought and let us know what you'd like to do. You can either float some ideas here (along the lines you already have), or [open up an issue](https://github.com/open-organization/open-org-leaders-manual/issues) outlining your contribution (I'll take care of triage; no worries there).

**Comment by LauraHilliger** [comment]
As it happens, I'm scheduled to speak with @amatlack in a few hours! We'll have a think about our co-authored piece and then update :) As always, huge shout out to @semioticrobotic for your continued enthusiasm and wrangling!

**Reply by semioticrobotic** [reply]
I thought I felt a fault line shift somewhere. Should have known you two were talking. Fair warning, @LauraHilliger: @amatlack is kind of a big shot now!

**Reply by LauraHilliger** [reply]
phh she was always a big shot!

**Reply by amatlack** [reply]
pfft taking advantage of a gal while she's ignoring GitHub notifications 😅 😆 

@LauraHilliger is going to look over our co-authored piece, as well as the old Tao piece. I reviewed the Tao piece when we first talked about it... it's a very early piece of my writing, and the quality is reflective of that, but I don't know what I would change. So I appreciate the extra 👀

**Reply by semioticrobotic** [reply]
Perfectly happy to have a different (or brand new!) piece from you in the book, @amatlack. I know I've essentially made publication of the Tao piece compulsory in the past because I like it so much, but I understand when it's time to move on. 😊

**Reply by amatlack** [reply]
I'm good with leaving it if you like it! 

I've only written three articles since we lost the site. Two might be a fit for a leaders' manual, if you want either or both (also happy to make edits if you have suggestions). I actually almost didn't post the first one I'm going to list because I didn't feel like I was saying anything new, but it was new to a lot of folks I shared it with. 

- https://amatlack.com/revisiting-employee-engagement/
- https://amatlack.com/pull-up-a-chair/

**Reply by semioticrobotic** [reply]
Honestly, whatever you want is fine with me! We should definitely be sure your pieces reflect your current thinking and represent work you're proud of. So give it a thought, let me know, and/or open up your issue(s)!

**Comment by jenkelchner** [comment]
Thanks, @semioticrobotic & everyone for picking this backup and keeping me in it :)  I'll review my chapters and give it a think over the next day or two!

**Comment by semioticrobotic** [comment]
Out sick today and seeing all the names of so many people I adore pop up in this thread really lifted my spirits. Looking forward to working with you all once again. Seems like we've already bypassed the "Is this something we want to do?" question and are on to the "How are we going to do this?" question. Fine with me. 😊

**Comment by semioticrobotic** [comment]
Since @shabnoorshah has already [opened an issue](https://github.com/open-organization/open-org-leaders-manual/issues/47) on this subject, I wanted to be sure she was aware of this conversation too.

**Comment by cll** [comment]
Hi Bryan,

I think both the CLL articles are good as written. Went through both to
check links, missing comments, and any post-covid necessary adjustments and
they are both good to go.

Thank you,

Catherine
p.s. I just did a workshop on comparing gaming strategy with product
strategy - the long-story-short of the workshop is collaboration wins over
competition. Would you like to see a draft of something like this, perhaps
titled "From Scrabble to Strategy: How Gaming Tactics Can Improve Your
Product Development"

**Reply by semioticrobotic** [reply]
Thanks for doing that review work, @cll. Really appreciate it.

Yes, of course I'm always happy to review new writing from you! If the piece on strategy (good for the Adaptability section!) is better reflective of your current thinking and pursuits, we might consider wapping it for one of the older pieces already in the book, just to freshen things up a bit. We can make that call later.

**Comment by HeidiHVL** [comment]
Thanks for initiating and coordinating this @semioticrobotic  I need to review my chapters, and might have time this weekend. My first impression is that I am not sure we should be bound by fitting things into a principle. More soon (crossing fingers).

**Comment by freedosproject** [comment]
I'd like to replace my "Open leadership lessons from a galaxy far, far away" essay with a more recent one, from Coaching Buttons. (Coaching Buttons is also CC-By.)

I think this one is a good candidate for the updated book:

[Leadership lessons from open source software](https://www.coachingbuttons.com/2024/02/19/lessonsopensource)

It's about 1,090 words.

@semioticrobotic

**Reply by semioticrobotic** [reply]
Nice! Thanks, @freedosproject. I'll give it a look.
